net.adoptopenjdk.v3.vanilla/src/main/java/net/adoptopenjdk/v3/vanilla/internal/AOV3AST.java

```diff
@@ -61,6 +61,13 @@
 
     @JsonProperty(value = "most_recent_lts", required = true)
     BigInteger mostRecentLTS = BigInteger.ZERO;
+
+    @JsonProperty(value = "most_recent_feature_version", required = true)
+    BigInteger getMostRecentFeatureVersion;
```

Comment:
```suggestion
    BigInteger getMostRecentFeatureVersion = BigInteger.ZERO;
```

Reply:
No need for that if the value is required, right?

Reply:
I believe that the current Jackson semantics are such that, if you specify a default value, it effectively turns off the check that requires the value to be present in the JSON.